player_linux.go

```diff
@@ -85,6 +86,10 @@ func newPlayer(sampleRate, numChans, bytesPerSample, bufferSizeInBytes int) (*pl
 	return &p, nil
 }
 
+func (p *player) SetUnderrunCallback(f func()) {
+	p.underrun = f
```

Comment:
Why do you think we need a Mutex? Other methods don't use a Mutex and require explicit synchronization if Player is used from multiple goroutines. Which it usually isn't.